docs/drafts/how-to-z-stream-upgrades.md

```diff
@@ -0,0 +1,23 @@
+## How-to: Z-Stream Automatic Updates
+
+To restrict automatic updates to only z-stream patches and avoid breaking changes, use the `"~"` version range operator when setting the version for the desired package in Catalog source.
+
+Example:
+
+```yaml
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterExtension
+metadata:
+  name: argocd
+spec:
+  installNamespace: argocd
```

Comment:
There was a recent [PR](https://github.com/operator-framework/operator-controller/pull/1200) that put the `installNamespace` and `serviceAccount` fields under an `install` root field.